src/components/Venue/menu/components/SendButton.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Box, Button, Grid, makeStyles } from '@material-ui/core'
```

Comment:
src/components/Venue/menu/components/SendButton.js
  Line 3:10:  'Box' is defined but never used   no-unused-vars
  Line 3:23:  'Grid' is defined but never used  no-unused-vars